<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskProcessControlWidget</class>
 <widget class="QWidget" name="TaskProcessControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>1042</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>1018</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>任务流程控制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ResultOutInfo">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="refreshBtn">
           <property name="text">
            <string> 刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="controlWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>360</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QTabWidget" name="ckTabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>S测控</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="CKTaskWorkModeTaskCodeSelectWidget" name="ckWidget1" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Ka测控</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="CKTaskWorkModeTaskCodeSelectWidget" name="ckWidget2" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Ka低速</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="SCTaskWorkModeTaskCodeSelectWidget" name="kadsWidget" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>X低速</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="SCTaskWorkModeTaskCodeSelectWidget" name="xdsWidget" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Ka高速</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="SCTaskWorkModeTaskCodeSelectWidget" name="kagsWidget" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="masterRadioButton">
              <property name="text">
               <string> 主用链路</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="slaveRadioButton">
              <property name="text">
               <string> 备用链路</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="linkSetBtn">
              <property name="text">
               <string> 配置宏下发</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="setParamMacroBtn">
              <property name="text">
               <string>参数宏下发</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sOneKeyXLBtn">
              <property name="text">
               <string>S校零</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="kaOneKeyXLBtn">
              <property name="text">
               <string>Ka校零</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="resourceReleaseBtn">
              <property name="text">
               <string>资源释放</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>测控</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>系统工作方式：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="systemWorkWay">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="setting">
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>设备组合</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>测控基带:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="macb"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>射频数字化前端:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="rfdf"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_17">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>确认</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="hpTabWidget">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>300</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="systemCaptureProcessControlGroupBox">
             <property name="title">
              <string>系统捕获流程控制</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13"/>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>高低速数传</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>系统工作方式：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="scsystemWorkWay">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="scsetting">
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="dshpatabWidget">
             <widget class="QWidget" name="dshpatabWidgetPage">
              <attribute name="title">
               <string>Ka低速数传功放</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_14"/>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="CCZFTaskPlanWidget" name="dtwidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKTaskWorkModeTaskCodeSelectWidget</class>
   <extends>QWidget</extends>
   <header>CKTaskWorkModeTaskCodeSelectWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SCTaskWorkModeTaskCodeSelectWidget</class>
   <extends>QWidget</extends>
   <header>SCTaskWorkModeTaskCodeSelectWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCZFTaskPlanWidget</class>
   <extends>QWidget</extends>
   <header location="global">CCZFTaskPlanWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
